camel/functions/search_functions.py

```diff
@@ -303,7 +303,61 @@ def search_google_and_summarize(query: str) -> str:
     return "Failed to find the answer from google search."
 
 
+def search_wolfram_alpha(query: str, detail: bool) -> str:
+    r"""Queries Wolfram|Alpha and returns the result. Wolfram|Alpha is a
```

Comment:
rename the var `detail` by `is_detailed` please

Reply:
OK, good advice.